documents/2-RTB/Documentazione_Esterna/Glossario.typ

```diff
@@ -0,0 +1,425 @@
+componenti dell’architettura seguendo quanto ideato dai progettisti; realizza gli strumenti per verificare e validare il
+codice; redige il Manuale Utente.
+
+== #LTG("Proof Of Concept (PoC)")
+
+Dimostrazione pratica per verificare la fattibilità o la validità di un'idea, di un concetto o di un progetto specifico.
+Nel contesto dello sviluppo software, un PoC consiste nella creazione di una versione semplificata di un'applicazione
+per testare un nuovo approccio tecnologico o per dimostrare la fattibilità di una funzionalità specifica.
+
+== #LTG("Pull Requests")
+
+Una Pull Request (PR) è una richiesta che un sviluppatore invia per proporre modifiche al codice di un progetto in un
+repository, tipicamente su piattaforme di versionamento del codice come GitHub, GitLab o Bitbucket. La PR permette ad
+altri membri del team o ai manutentori del progetto di revisionare il codice proposto prima che venga integrato nel ramo
+principale (di solito chiamato main o master).
+
+#pagebreak()
+
+= #LTG("Q")
+#pagebreak()
+
+= #LTG("R")
+
+== #LTG("Repository")
+
+Un repository (o repo) è un archivio digitale che contiene file, cartelle e informazioni relative al progetto, inclusi
+il codice sorgente, la documentazione e la cronologia delle modifiche. È uno spazio organizzato, solitamente gestito
+tramite un sistema di controllo versione come Git, dove i file sono archiviati e tenuti sotto controllo, permettendo di
+monitorare le modifiche nel tempo. I repository sono usati per gestire il ciclo di vita di un progetto software,
+facilitando la collaborazione tra sviluppatori e il versionamento del codice. I repository possono essere locali (sul
+proprio computer) o remoti (su piattaforme come GitHub, GitLab, Bitbucket).
+
+== #LTG("RTB (Requirements and Technology Baseline)")
+
+Fase iniziale e fondamentale del processo di sviluppo di un software. In questa fase, l'obiettivo principale è stabilire
+e comprendere i requisiti del sistema e definire la base tecnologica sulla quale il progetto si svilupperà. Le tre
+principali attività di questa fase sono: l'analisi dei requisiti, la definizione della baseline tecnologica e la
+definizione della baseline di progetto
+
+== #LTG("Responsabile")
+
+Figura fondamentale che coordinare il gruppo, funge da punto di riferimento per il committente e per il team, svolgendo
+il ruolo di mediatore tra le due parti. In particolare si occupa di: gestire le relazioni con l’esterno; pianificare le
+attività (quali svolgere, data di inizio e fine, assegnazione delle priorità); valutare i rischi delle scelte da
+effettuare; controllare i progressi del progetto; gestire le risorse umane; approvare la documentazione
+
+#pagebreak()
+
+= #LTG("S")
+
+== #LTG("Stato Avanzamento Lavori (SAL)")
+
+Nell'ambito del project management, è una riunione periodica (o attività di confronto analoga) che viene stabilita per
+garantire e verificare l'avanzamento di un progetto rispetto ai propri obiettivi
```

Comment:
metterei in riga 317 "rispetto agli obbiettivi prestabiliti" (al posto di "propri obbiettivi")